### PR TITLE
Fixes #18400: Add release notes for 6.2

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/18400

### CODE REVIEW

File: release-data/changelogs/6.2/index.adoc [hunk 1]
```diff
@@ -2,6 +2,49 @@
...
+* *Automated plugin upgrade*. No more incompatible or disabled plugins after upgrade! (if your server
+  has access to the download server).
+* Rudder *instances behind proxies* will now be able to download plugins and use external APIs through
+  your proxy.
+
+=== Platforms
+
+* *Windows* agent is now capable of applying DSC configurations directly. You can either use a source config
+  that the agent will compile and apply, or directly apply a compiled state (in MOF format). In both cases
+  the outcome will be reported in Rudder like any other component.
```
Comment (fanf) [inline release-data/changelogs/6.2/index.adoc:39]: Talk about chocolatey support too ?

Comment (fanf) [reply release-data/changelogs/6.2/index.adoc:39]: I think we need to talk about support for relay in docker container

File: release-data/changelogs/6.2/index.adoc [hunk 2]
```diff
@@ -2,6 +2,49 @@
...
+* An *API* endpoint for precise monitoring
+* *Dedicated page* for easy visualization
+* *Notifications* in the menu bar not to miss important stuff
+
+== Plugins
+
+* *Automated plugin upgrade*. No more incompatible or disabled plugins after upgrade! (if your server
+  has access to the download server).
+* Rudder *instances behind proxies* will now be able to download plugins and use external APIs through
+  your proxy.
```
Comment (fanf) [inline release-data/changelogs/6.2/index.adoc:33]: I think we need to talk about plugin CVE here or in a dedicated paragraph: version 1.0, etc

Comment (amousset) [reply release-data/changelogs/6.2/index.adoc:33]: 1.0 was released with Rudder 6.1, but the interface changed a bit with the view by node, we can talk about it indeed.

**Comment by amousset** [comment]
PR updated with a new commit

**Review by fanf — APPROVED** [review]

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**